rustls/src/msgs/deframer.rs

```diff
@@ -492,9 +492,9 @@ impl DeframerBuffer<'_, InternalPayload> for DeframerVecBuffer {
 }
 
 impl<'a> DeframerBuffer<'a, ExternalPayload<'a>> for DeframerVecBuffer {
-    fn copy(&mut self, payload: &ExternalPayload<'a>, at: usize) {
+    fn copy(&mut self, payload: &ExternalPayload<'a>, _at: usize) {
         let len = payload.len();
-        self.unfilled()[at..at + len].copy_from_slice(payload.0);
+        self.unfilled()[0..len].copy_from_slice(payload.0);
```

Comment:
Nit: would be idiomatic to just leave off the `0` in this range expression.

Reply:
Fixed.